airflow/providers/tableau/CHANGELOG.rst

```diff
@@ -24,6 +24,21 @@
 Changelog
 ---------
 
+3.1.0
+.....
+
+Features
+~~~~~~~~
+
+* ``Remove Tableau from Salesforce provider (#23747)``
```

Comment:
I think we still have to release because there was a code change in this commit. 
However I moved it to 3.0.1 and changed it to "bugfix" - part of #23747 was to change "operator" to "module" in warning message, so we HAVE TO release it if we want this message to be updated. But it is indeed not a feature but bugfix.

Reply:
We have to have "SOME" change in the changelog. We have a few others that are rather "Misc" changes than bugfix and maybe in the future we should add it.